src/commands/targeting/update.ts

```diff
@@ -117,5 +121,18 @@ export default class UpdateTargeting extends Base {
             featureConfig
         )
         this.writer.showResults(result)
+        this.showSuggestedCommand(featureKey, envKey, result, isTargetingUpdated)
+    }
+
+    private showSuggestedCommand(featureKey: string, envKey: string, result: FeatureConfig, isTargetingUpdated: boolean = true) {
+        const { status, targets } = result
+        if (status === 'active' || targets.length === 0)  {
+            return
+        }
+        const message = (isTargetingUpdated ? '\nTargeting rules updated.\n' : '') +
```

Comment:
It seems a bit excessive to do a deep comparison of the feature configs just to print this message. Considering updating targeting rules is the only thing this command does, and at this point the request has been successfully sent whether the payload changed or not, could we either always show this or just omit it entirely?

Reply:
hmm...should we just omit it then? caz I was trying not to show this line for continue and exit prompt, don't want it be a misleading message. and you're right, I feel like the comparison is excessive too...